app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -1243,6 +1243,31 @@ class BrowserTabViewModelTest {
         assertSame(authenticationRequest, requiresAuthCommand.request)
     }
 
+    @Test
+    fun whenAuthenticationIsRequiredForSameHostThenNoChangesOnBrowser() {
+        val mockHandler = mock<HttpAuthHandler>()
+        val siteURL = "http://example.com/requires-auth"
+        val authenticationRequest = BasicAuthenticationRequest(mockHandler, "example.com", "test realm", siteURL)
+
+        loadUrl(url = "http://example.com", isBrowserShowing = true)
+        testee.requiresAuthentication(authenticationRequest)
+
+        assertEquals("http://example.com", omnibarViewState().omnibarText)
```

Comment:
Since the opposite test (the next one) checks that that the `HideWebViewContent` command *is* sent does it make sense to check that the `HideWebViewContent` command *is not* sent here?

Reply:
Sounds good 👍